web/templates/shared/_form_errors.html.eex

```diff
@@ -1,8 +1,8 @@
 <%= if display_errors?(@form) do %>
   <div class="alert alert-danger">
   <p>Oops, something went wrong! Please check the errors below:</p> <ul>
-  <%= for {attr, message} <- @form.errors do %>
-    <li><%= humanize(attr) %> <%= message %></li>
-  <% end %> </ul>
+    <%= for {key, {value, _}} <- @form.errors do %>
```

Comment:
Could you explain what this is doing differently? I haven't done much with form errors so I'm not sure what's going on with this. I'd love to learn more :)